.github/labeler.yml

```diff
@@ -0,0 +1,201 @@
+Docs:
+  - docs/*
+  - docs/**/*
+  - examples/*
+  - examples/**/*
+  - licenses/*
+  - CITATION
+  - .mailmap
+  - .readthedocs.yml
+  - '*.md'
+  - '*.rst'
+
+testing:
+  - astropy/tests/*
+  - astropy/tests/**/*
+  - astropy/*/tests/*
+  - astropy/**/tests/*
+  - codecov.yml
+  - conftest.py
+  - '*/conftest.py'
+  - '*/**/conftest.py'
+  - azure-pipelines.yml
+  - tox.ini
+  - .circleci/*
+  - .github/*.yml
+  - .github/workflows/*.yml
+  - .pep8speaks.yml
+  - .pyinstaller/*
+  - .pyinstaller/**/*
+  - .travis.yml
+
+external:
+  - astropy/extern/*
+  - astropy/extern/**/*
+  - cextern/*
+  - cextern/**/*
+
+installation:
+  - docs/install.rst
+  - MANIFEST.in
+  - pip-requirements
+  - pyproject.toml
+  - setup.*
+
+Release:
+  - docs/development/releasing.rst
+
+config:
+  - astropy/config/*
+  - astropy/config/**/*
+  - docs/config/*
+
+constants:
+  - astropy/constants/*
+  - astropy/constants/**/*
+  - docs/constants/*
```

Comment:
Can we combine these three entries? At least we could do ``*/constants/*`` for the first right? (then don't need docs one)

Reply:
Yes, after doing some testing `'**/constants/**/*'` seems to work.